package.json

```diff
@@ -12,7 +12,8 @@
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test",
-    "test-single-run": "ng test --single-run",
+    "ci:build": "ng build --prod --no-progress",
+    "ci:test": "ng test --single-run --code-coverage --no-progress",
     "release": "ng build --target production --build-optimizer",
```

Comment:
We may want to have some different commands for building the demo app, e.g. `ci:build` and commands for building the library. Right now the `npm run package` command will run `node build.js` and then to publish the library to npm you would run `npm publish dist` with whatever prerelease tags we want to set.

Reply:
I agree with that. I thought I remembered that people had agreed to wanting to do the npm packaging manually, at least at first. That would be an easy addition that we could do later, but want I wanted to achieve immediately was better info for reviewers on whether or not a pull request passes the basic set of tests that we have.